sdk/storage/azure-storage-file/azure/storage/file/file_service_client.py

```diff
@@ -270,11 +272,10 @@ def list_shares(
             self, name_starts_with=None,  # type: Optional[str]
             include_metadata=False,  # type: Optional[bool]
             include_snapshots=False, # type: Optional[bool]
-            marker=None,
             timeout=None,  # type: Optional[int]
             **kwargs
         ):
-        # type: (...) -> SharePropertiesPaged
+        # type: (...) -> ItemPaged[ShareProperties]
```

Comment:
It seems various return type hints are used throughout - in some cases it's `Iterable[IterType]`, sometimes `ItemPaged[IterType]` and sometimes just `ItemPaged`.

Reply:
That probably should be Iterator to be more accurate without being too much specific

Reply:
Actually, it should be ItemPaged, since this guy has a "by_page" that should be documented and accessible

Reply:
Fixed them all